src/main/java/com/puppycrawl/tools/checkstyle/utils/ScopeUtil.java

```diff
@@ -34,16 +34,17 @@ private ScopeUtil() {
     }
 
     /**
-     * Returns the Scope specified by the modifier set.
+     * Returns the Scope specified by the modifier set. If no modifiers are present,
+     * the scope is assumed to be public an interface or annotation block and
+     * package protected otherwise.
      *
      * @param aMods root node of a modifier set
      * @return a {@code Scope} value
      */
     public static Scope getScopeFromMods(DetailAST aMods) {
-        // default scope
```

Comment:
Please share grep on source code to prove that only 2 Check use this method.

Reply:
You are right, this method is used by another utility method `getSurroundingScope`. We need test inputs for all of them.

Reply:
Please share details of dependencies on this method, we might need to run regression on all affected.